projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.html

```diff
@@ -30,13 +30,13 @@
           {{ 'Is the query hearing related?' | rpxTranslate }}
         </th>
         <td class="govuk-table__cell" [class.govuk-table__cell--no-border]="query.isHearingRelated">
-          {{ 'Is the query hearing related?' | rpxTranslate: null : (query.isHearingRelated ? 'Yes' : 'No') }}</td>
+          {{ 'Is the query hearing related?' | rpxTranslate: null : (query.isHearingRelated) }}</td>
       </tr>
-      <tr class="govuk-table__row govuk-table__row--isHearingRelated" *ngIf="query.isHearingRelated">
+      <tr class="govuk-table__row govuk-table__row--isHearingRelated" *ngIf="query.isHearingRelate === 'Yes'">
```

Comment:
possible typo "query.isHearingRelate" instead of isHearingRelated

Reply:
That's updated